controllers/events/event.go

```diff
@@ -61,29 +61,36 @@ const (
 )
 
 type EventSourceConfig struct {
-	EventSourceComponentName string `json:"eventSourceComponentName"`
-	EventSourceTopic         string `json:"eventSourceTopic,omitempty"`
-	EventBusComponentName    string `json:"eventBusComponentName,omitempty"`
-	EventBusTopic            string `json:"eventBusTopic,omitempty"`
-	SinkComponentName        string `json:"sinkComponentName,omitempty"`
-	EventSourceSpecEncode    string `json:"eventSourceSpecEncode,omitempty"`
-	EventBusSpecEncode       string `json:"eventBusSpecEncode,omitempty"`
-	SinkSpecEncode           string `json:"sinkSpecEncode,omitempty"`
+	EventSourceComponent  string `json:"eventSourceComponent"`
+	EventSourceTopic      string `json:"eventSourceTopic,omitempty"`
+	EventBusComponent     string `json:"eventBusComponent,omitempty"`
+	EventBusTopic         string `json:"eventBusTopic,omitempty"`
+	SinkComponent         string `json:"sinkComponent,omitempty"`
+	EventSourceSpecEncode string `json:"eventSourceSpecEncode,omitempty"`
+	EventBusSpecEncode    string `json:"eventBusSpecEncode,omitempty"`
+	SinkSpecEncode        string `json:"sinkSpecEncode,omitempty"`
 }
 
 type TriggerConfig struct {
-	EventBusComponentName string               `json:"eventBusComponentName"`
-	EventBusTopics        []string             `json:"eventBusTopics,omitempty"`
-	Subscribers           []*SubscriberConfigs `json:"subscribers,omitempty"`
-	EventBusSpecEncode    string               `json:"eventBusSpecEncode,omitempty"`
-	SinkSpecEncode        string               `json:"sinkSpecEncode,omitempty"`
+	EventBusComponent  string                 `json:"eventBusComponent"`
+	Inputs             []*Input               `json:"inputs,omitempty"`
```

Comment:
needn't be the same as the previous modification `map[string]*Input`?

Reply:
It is the same struct as in events-handlers